packages/augur-ui/src/modules/market/components/market-view/market-view.tsx

```diff
@@ -159,7 +159,7 @@ export default class MarketView extends Component<
 
     const {
       isConnected,
-      loadFullMarket,
+      loadMarketsInfo,
```

Comment:
Is that the specific info for a *single* market? If so, I suggest `loadMarketInfo`, singular.

Reply:
that is the method name used over the app. going to leave it.

Reply:
also it take an array